README.md

```diff
@@ -183,7 +183,7 @@ You can get the center coordinate of the hexagon like so:
 ./bin/cellToLatLng --index 8a2a1072b59ffff
 ```
 
-This will produce some coordinate:
+This will produce some coordination:
```

Comment:
This is not a typo. The following two numbers are a [coordinate](https://www.merriam-webster.com/dictionary/coordinate#dictionary-entry-3)